src/content/app/entity-viewer/variant-view/variant-view-sidebar/overview/VariantOverview.tsx

```diff
@@ -0,0 +1,129 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+
+import useEntityViewerIds from 'src/content/app/entity-viewer/hooks/useEntityViewerIds';
+import { useDefaultEntityViewerVariantQuery } from 'src/content/app/entity-viewer/state/api/entityViewerThoasSlice';
+
+import variantGroups from 'src/content/app/genome-browser/constants/variantGroups';
+
+import MainAccordion from './MainAccordion';
+import { VariantDB } from 'src/content/app/genome-browser/components/drawer/drawer-views/variant-summary/VariantSummary';
+import VariantConsequence, {
+  getMostSevereVariantConsequence
+} from 'src/content/app/genome-browser/components/drawer/drawer-views/variant-summary/variant-consequence/VariantConsequence';
+import ExternalLink from 'src/shared/components/external-link/ExternalLink';
+
+import styles from './VariantOverview.module.css';
+
+const VariantOverview = () => {
+  const { activeGenomeId, parsedEntityId } = useEntityViewerIds();
+
+  const { objectId: variantId } = parsedEntityId ?? {};
+
+  const { currentData, isFetching } = useDefaultEntityViewerVariantQuery(
+    {
+      genomeId: activeGenomeId ?? '',
+      variantId: variantId ?? ''
+    },
+    {
+      skip: !activeGenomeId || !variantId
+    }
+  );
+
+  if (isFetching) {
+    return <div>Loading...</div>;
+  }
+
+  if (!currentData?.variant) {
+    return <div>No data to display</div>;
+  }
+
+  const { variant } = currentData;
+
+  const mostSevereConsequence = <VariantConsequence variant={variant} />;
+
+  const variantConsequence = getMostSevereVariantConsequence(variant);
+
+  if (!variantConsequence) {
+    return null;
+  }
+
+  const currentVariantGroup = variantGroups.find(({ variant_types }) =>
+    variant_types.find(({ label }) => label === variantConsequence)
+  );
+
+  const currentVariantType = currentVariantGroup?.variant_types.find(
+    ({ label }) => label === variantConsequence
+  );
```

Comment:
The label for the group is plural (e.g. "Protein altering variants", "Transcript variants"). We will want it in a singular form.

_(Not a blocker for this PR)_

Reply:
Done, updated variant types file.